config.yaml

```diff
@@ -3,3 +3,15 @@ options:
     type: string
     default: internet
     description: Data Network Name (DNN)
+  amf-ip:
```

Comment:
And couldn't we simply have url to encompass both the ip and hostname? It's a bit strange to have both

Reply:
If we update the `fiveg_n2` to take url would that be ok? That url could be a hostname or an IP.